### ISSUE TITLE
Bootstrapper crashed with signal 11 Corellium

### ISSUE THREAD

**Issue by AeonLucid** [frida/frida#2744]
Starting from Frida 16.1.9 and up, Frida no longer functions on Corellium Android 10 arm64.
This is a crash report from 16.1.9.

```
bootstrapper crashed with signal 11 at offset 0x1514; please file a bug
	 pc: 00000070225b0514
	 lr: 00000070225afab4
	 sp: 0000007fd290fb00
	 x0: 0000006f94079c20
	 x1: 00000070225b0b91
	 x2: 0000000000000001
	 x3: 0000000000000002
	 x4: 000000000000001b
	 x5: 0000000000000000
	 x6: 0000007023eb3000
	 x7: 0000000000000000
	 x8: 0000000000000002
	 x9: 0000000000000002
	x10: 0000000000000001
	x11: 0000006f94078448
	x12: 0000000000000001
	x13: 0000000000000001
	x14: 0000000000000000
	x15: 00004d07a77190d7
	x16: 0000006f964b3038
	x17: 000000701ebd130c
	x18: 0000007023336000
	x19: 0000007fd290fbb8
	x20: 0000007fd290fb38
	x21: 0000006f94079c20
	x22: 000000000000001b
	x23: 0000000000185488
	x24: 0000006f93ef3000
	x25: 0000006f94077de2
	x26: 000000000000252d
	x27: 0000006f94002558
	x28: 431bde82d7b634db
	x29: 0000007fd290fb00
```

**Comment by dem0ns** [comment]
#2738

**Comment by oleavr** [comment]
Fixed in latest git. Will be available in 16.1.11.
